modules/upgrade/pages/client-intro.adoc

```diff
@@ -0,0 +1,10 @@
+[[client-migration]]
+= Upgrade the Clients
+
+
+Clients use the versioning system of their underlying operating system.
+For most clients using {suse} operating systems, you can perform minor upgrade within the {productname} {webui}.
+
+To upgrade clients running SLE{nbsp}12 to SLE{nbsp}15, the upgrade is automated, but you will need to do some preparation steps before you begin.
```

Comment:
Maybe, we could generalize this?  It's major version upgrade and should also be valid for Leap upgrades.

Reply:
Yes, agreed